workflow/alerting/setup-alert.mdx

```diff
@@ -1,68 +1,69 @@
 ---
-title: Setup Alerts
+title: Creating Alerts
 ---
 
-# Overview
-Alerts let you set up customizable alert policies for anything that you can monitor. Receive notifications for fluctuations in key performance metrics as data streams in from all of our products, including APM, infrastructure, and Kubernetes.
+The Alerts page is a tool for monitoring and responding to critical events across your entire infrastructure. Create high-level alerts to detect system-level anomalies or supervise a single Kubernetes node. Set up alerts using the OpenTelemetry data your team collects to effectively manage system-wide issues.
 
-Setup alerts
-------------
-Alerts can be configured to notify you when certain conditions are met, such as when a metric value exceeds a certain threshold or when a log event occurs that matches a specific pattern. In addition, alerts can be configured to send notifications to Slack or Gmail, making it easier to stay informed about critical events.
+# How Alerts Work
 
--   Navigate to "Alerts" from the left menu
--   Go to the "Rules" tab and click on "Create new"
+Alerts can be created with __metric__, __log__, or __trace__ data. Log and trace alerts are triggered when the MW Backend Alerts Server (which has already fetched the pre-defined rules specified by the user) identifies that the total number (count) of logs or traces have fired within the pre-defined time window. 
 
-### Select Monitor type
+Metric Alerts are triggered when the pre-defined rules that the user specifies are satisfied at the exact time that the MW Backend Alerts Server checks the metrics endpoint (e.g. If your window is set to five minutes, the MW Backend Alerts Server checks the Metrics endpoint if the conditions are met once every five minutes).
 
-Select the Monitor type corresponding to the kind of alert you want to receive,
+<Warning> The MW Backend Alerts Server is not checking your pre-defined alert conditions in real time. </Warning>

```

Comment:
Let's remove this. Most users don't ask it and it can accidentally disuade users who don't really understand most alerting systems are not real time. The paragraph above does good already describing that we trigger at the time frame set.